indexer/feature_utils.hpp

```diff
@@ -12,10 +12,10 @@ namespace feature
 {
   static constexpr uint8_t kMaxStarsCount = 7;
   static constexpr std::string_view kFieldsSeparator = " • ";
-  static constexpr std::string_view kToiletsSymbol = "🚻";
-  static constexpr std::string_view kAtmSymbol = "💳";
-  static constexpr std::string_view kWheelchairSymbol = "♿️";
-  static constexpr std::string_view kWifiSymbol = "🛜";
+  static constexpr std::string_view kToiletsSymbol = "🚻︎";
+  static constexpr std::string_view kAtmSymbol = "💳︎";
+  static constexpr std::string_view kWheelchairSymbol = "♿";
+  static constexpr std::string_view kWifiSymbol = "🛜︎";
```

Comment:
@Misalf-git PTAL

Reply:
Toilet, ATM and Wifi have vs15.
But Wheelchair has no selector.